data/sssd-tests/ldap/test.sh

```diff
@@ -14,7 +14,9 @@ mkdir -p /tmp/ldap-sssdtest &&
 cp ldap.crt /tmp/ldap-sssdtest.cacrt &&
 cp ldap.crt /tmp/ldap-sssdtest.crt &&
 cp ldap.key /tmp/ldap-sssdtest.key &&
-/usr/sbin/slapd -h 'ldap:///' -f slapd.conf &&
+SLAPD="/usr/sbin/slapd"
+[ ! -e "$SLAPD" ] && SLAPD=/usr/lib/openldap/slapd
+$SLAPD -h 'ldap:///' -f slapd.conf &&
```

Comment:
[DRY](https://en.wikipedia.org/wiki/Don't_repeat_yourself). You are calling the same code multiple times, please [extract method](http://refactoring.com/catalog/extractMethod.html).